src/cascade/model/grids.py

```diff
@@ -0,0 +1,174 @@
+"""Support for dismod concepts that are expressed over grids: age-time, smoothings and value priors.
+"""
+from math import isclose, isinf
+
+GRID_SNAP_DISTANCE = 1 / 365
+
+
+class AgeTimeGrid:
+    """Structure for storing age and times over which the model will estimate.
+    Also constructors for making standard age-time grids.
+    """
+
+    @classmethod
```

Comment:
I was reading that one purpose of using a class method involves inheritance.  If a class inherits from AgeTimeGrid,  and uniform() is called, that class will be returned rather than AgeTimeGrid.  I will read some more, just wondering as a I read through.

Reply:
That's correct. I don't expect this to be used that way. I'm actually not sure why I didn't make it a separate factory function, that's what I'd normally do. I'm imagining that there might be several of these alternate constructors which is why I thought to attach it to the class. Dunno. Many ways to skin cats etc.